awd/cli.py

```diff
@@ -181,38 +184,42 @@ def deposit(
     try:
         s3_client.client.list_objects_v2(Bucket=bucket)
     except ClientError as e:
-        logger.error(
-            f"Error accessing bucket: {bucket}, {e.response['Error']['Message']}"
+        logger.exception(
+            "Error accessing bucket: %s, %s", bucket, e.response["Error"]["Message"]
         )
-        exit
+        sys.exit()
```

Comment:
I might propose just using `return` to break out of this CLI command which returns nothing.  `sys.exit()` is a bit more abrupt and might not handle any cleanup that click or other libraries would perform.